### ISSUE TITLE
display forks stars and contributors,admin of this repo by creating a footer at the base page of knowledgeshare

### ISSUE THREAD

**Issue by tejaswini22199** [tejaswini22199/KnowledgeShare#40]

**Comment by Navpreet1234** [comment]
Plz assign this issue to me.

**Comment by Navpreet1234** [comment]
![Screenshot (72)](https://user-images.githubusercontent.com/47537671/102878064-b395e000-446d-11eb-9181-2f8479aec9d4.png)
Is this footer fine??

**Comment by tejaswini22199** [comment]
mention number of forks no of stars and contributors , add link to contributors and show number of contributors just write admin it is enough no need of writing admin of repo. Please do these changes and make footer height 50% of this

**Comment by Navpreet1234** [comment]
![Screenshot (73)](https://user-images.githubusercontent.com/47537671/102900182-efdb3780-4491-11eb-93b4-9712419c96c0.png)
I have already added the link to contributors and admin and also if someone wants to see the forks and stars he can click the buttons respectively.

**Comment by tejaswini22199** [comment]
can you remove all github ids just mention the number of contributors because it is becoming large as contributors increase

**Comment by Navpreet1234** [comment]
Is the rest part fine??

**Comment by Navpreet1234** [comment]
Is the rest part fine??

**Comment by tejaswini22199** [comment]
> Is the rest part fine??

yeah! And make sure you are in upstream with main branch so that there are no conflicts present.
